sbc_compassion/models/correspondence_template.py

```diff
@@ -67,23 +67,27 @@ class CorrespondenceTemplate(models.Model):
         help='Y Position of the upper-right corner of the bluesquare '
              'in pixels')
     qrcode_x_min = fields.Integer(
+        compute="_compute_size",
```

Comment:
It is a shortcut to this field (and, as the value is stored in %, avoid to compute it everytime).
I don't know if it is useful to have it visible, but I prefer to keep it inside the model.